policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/credits/ar_additional_credit.yaml

```diff
@@ -0,0 +1,148 @@
+  period: 2022
+  input:
+    state_code: AR
+    is_tax_unit_head_or_spouse: true
+    ar_files_separately: true
+    ar_taxable_income_indiv: 25_101
+    ar_taxable_income_joint: 25_101
+  output: 
+    ar_additional_credit: 15
+
+- name: One person with higher income
+  period: 2022
+  input:
+    state_code: AR
+    is_tax_unit_head_or_spouse: true
+    ar_files_separately: true
+    ar_taxable_income_indiv: 50_000
+    ar_taxable_income_joint: 50_000
+  output: 
+    ar_additional_credit: 0
+
+- name: Two people filing separately with no income
+  period: 2022
+  input:
+    people:
+      person1:
+        is_tax_unit_head_or_spouse: true
+        ar_taxable_income_indiv: 0
+        ar_taxable_income_joint: 0
+      person2:
+        is_tax_unit_head_or_spouse: true
+        ar_taxable_income_indiv: 0
+        ar_taxable_income_joint: 0
+    tax_units:
+      tax_unit:
+        members: [person1, person2]
+        ar_files_separately: true
+    households:
+      household:
+        members: [person1, person2]
+        state_code: AR
+  output: 
+    ar_additional_credit: [60, 60]
+
+- name: Two people filing separately with one high income and one low income
+  period: 2022
+  input:
+    people:
+      person1:
+        is_tax_unit_head_or_spouse: true
+        ar_taxable_income_indiv: 25_100
+        ar_taxable_income_joint: 25_100
+      person2:
+        is_tax_unit_head_or_spouse: true
+        ar_taxable_income_indiv: 50_000
+        ar_taxable_income_joint: 50_000
+    tax_units:
+      tax_unit:
+        members: [person1, person2]
+        ar_files_separately: true
+    households:
+      household:
+        members: [person1, person2]
+        state_code: AR
+  output: 
+    ar_additional_credit: [20, 0]
+
+- name: Two people filing jointly with middle income
+  period: 2022
+  input:
+    people:
+      person1:
+        is_tax_unit_head_or_spouse: true
+        ar_taxable_income_indiv: 0
+        ar_taxable_income_joint: 25_100
+      person2:
+        is_tax_unit_head_or_spouse: false
+        ar_taxable_income_indiv: 0
+        ar_taxable_income_joint: 0
+    tax_units:
+      tax_unit:
+        members: [person1, person2]
+        ar_files_separately: false
+    households:
+      household:
+        members: [person1, person2]
+        state_code: AR
+  output: 
+    ar_additional_credit: [40,0]
+
+- name: Two people filing jointly with no income
+  period: 2022
+  input:
+    people:
+      person1:
+        is_tax_unit_head_or_spouse: true
+        ar_taxable_income_indiv: 0
+        ar_taxable_income_joint: 0
+      person2:
+        is_tax_unit_head_or_spouse: false
+        ar_taxable_income_indiv: 0
+        ar_taxable_income_joint: 0
+    tax_units:
+      tax_unit:
+        members: [person1, person2]
+        ar_files_separately: false
+    households:
+      household:
+        members: [person1, person2]
+        state_code: AR
+  output: 
+    ar_additional_credit: [120,0]
```

Comment:
can you create a test with head and spouse - filing jointly and both having joint income that would give them the max credit

Reply:
Hi Pavel, I'm a little confused about what you meant by "head and spouse", is this the same as married couple filling jointly?

Reply:
Yes, basically a test where two people get $120